docs/documentation.md

```diff
@@ -33,22 +35,63 @@ $> rake db:migrate
 $> rails server
 ```
 
-Visit `http://localhost:3000/admin` and log in using:
+Visit `http://localhost:3000/admin` and you'll be on your brand
+new Active Admin dashboard.
 
-* __User__: admin@example.com
-* __Password__: password
+To register an already existing model, run:
 
-Voila! You&#8217;re on your brand new Active Admin dashboard.
```

Comment:
I just noticed that the "Voila!", and the HTML characters were already there, thanks for updating that even if it was not introduced by you :+1:.

Reply:
:-)